esmvalcore/config-user.yml

```diff
@@ -36,19 +36,21 @@ profile_diagnostic: false
 # Rootpaths to the data from different projects (lists are also possible)
 # these are generic entries to better allow you to enter your own
 # For site-specific entries, see below
-# rootpath:
-#   CMIP5: [~/cmip5_inputpath1, ~/cmip5_inputpath2]
-#   OBS: ~/obs_inputpath
-#   RAWOBS: ~/rawobs_inputpath
-#   default: ~/default_inputpath
-#   CORDEX: ~/default_inputpath
+# Comment out these when using a site-specific path
+rootpath:
```

Comment:
This was commented out because we got the complaint from users that these default settings do not work ever for anyone.